R/db.R

```diff
@@ -279,107 +279,97 @@ setMethod("sqlCreateTable", "DB2/AIX64",
 
 # Microsoft SQL Server ---------------------------------------------------------
 
+#' SQL Server
+#'
+#' Details of SQL Server specific variations for odbc and DBI generics.
+#'
 #' @rdname SQLServer
 #' @usage NULL
 setClass("Microsoft SQL Server", where = class_cache)
 
-#' SQL Server specific implementation.
-#'
-#' For SQL Server, `conn@quote` will return the quotation mark, however
-#' both quotation marks as well as square bracket are used interchangeably for
-#' delimited identifiers.  See:
-#' \url{https://learn.microsoft.com/en-us/sql/relational-databases/databases/database-identifiers?view=sql-server-ver16}
-#' Therefore strip the brackets first, and then call the DBI method that strips
-#' the quotation marks.
-#' TODO: the generic implementation in DBI should take a quote char as
-#' parameter.
+#' @description
+#' ## `dbUnquoteIdentifier()`
 #'
+#' `conn@quote` returns the quotation mark, but quotation marks and square
+#' brackets can be interchangeably for delimited identifiers.
+#' (<https://learn.microsoft.com/en-us/sql/relational-databases/databases/database-identifiers>).
+#' So this function strips the brackets first, and then calls the DBI method to
+#' strip the quotation marks.
 #' @rdname SQLServer
 #' @docType methods
-#' @inheritParams DBI::dbUnquoteIdentifier
 #' @usage NULL
```

Comment:
We'll be able to remove `@usage NULL` once dev roxygen2 is out. I made issue #666 (👿) to remember.

Reply:
😈👹👿